### ISSUE TITLE
Question: Custom Plugin Search

### ISSUE THREAD

**Issue by LaocheXe** [e107inc/e107#3918]
Plugin Manager generated Admin Area for a custom plugin - I've gone through the code and see no where how to search/filter the details on the page.

I got plugin that has a  few things like (user_id (but shows user_name)), unit id number, rank, etc... I can't search for user names, if I search their unit id number - it displays.

Basically I'd like to expand the functionality of the search, to include user name, and some others. Any idea how to do that?

**Comment by Moc** [comment]
Is it this plugin? https://github.com/LaocheXe/RSystems?

**Comment by LaocheXe** [comment]
Yes it  is - Admin Area > Roster System > Service Records - Add some  records, then try to search user name, or others.
